tests/WE2E/test_configs/wflow_features/config.long_fcst.yaml

```diff
@@ -0,0 +1,26 @@
+metadata:
+  description: |-
+    This test checks the capability of the workflow to retrieve from NOAA
```

Comment:
@mkavulich Please add in the description that this test is also to check tha the weather model can perform a relatively long forecast.

Reply:
Done